build/client/run.sh

```diff
@@ -50,72 +50,72 @@ PKG_NAME=df-client
 
 pre() {
     echo "PRE: clean and create ${BIN_DIR}"
-    createDir ${BIN_DIR}
+    createDir "${BIN_DIR}"
 }
 
 check() {
-    cd ${BUILD_SOURCE_HOME}
+    cd "${BUILD_SOURCE_HOME}"
     exclude="vendor/"
 
     # gofmt
     echo "CHECK: gofmt, check code formats"
-    result=`find . -name '*.go' | grep -vE "${exclude}" | xargs gofmt -s -l -d 2>/dev/null`
+    result=$(find . -name '*.go' | grep -vE "${exclude}" | xargs gofmt -s -l -d 2>/dev/null)
     [ ${#result} -gt 0 ] && (echo "${result}" \
         && echo "CHECK: please format Go code with 'gofmt -s -w .'" && false)
 
     # golint
     which golint > /dev/null || export PATH=${BUILD_GOPATH}:$PATH
     which golint > /dev/null || (echo "CHECK: install golint" \
         && go get -u golang.org/x/lint/golint; \
-            cp ${BUILD_GOPATH}/bin/golint ${BUILD_GOPATH}/)
+            cp "${BUILD_GOPATH}/bin/golint" "${BUILD_GOPATH}"/)
 
     echo "CHECK: golint, check code style"
-    result=`go list ./... | grep -vE "${exclude}" | sed 's/^_//' | xargs golint`
+    result=$(go list ./... | grep -vE "${exclude}" | sed 's/^_//' | xargs golint)
     [ ${#result} -gt 0 ] && (echo "${result}" && false)
 
     # go vet check
     echo "CHECK: go vet, check code syntax"
-    packages=`go list ./... | grep -vE "${exclude}" | sed 's/^_//'`
-    go vet ${packages} 2>&1
+    packages=$(go list ./... | grep -vE "${exclude}" | sed 's/^_//')
+    go vet "${packages}" 2>&1
```

Comment:
The `"${packages}"` will be treated as one whole string parameter by `go vet`. Actually we want `go vet` receives all files not one file. A workaround is:

```bash
echo "${packages}" | xargs go vet 2>&1
```